src/platform/utilities/feature-toggles/index.jsx

```diff
@@ -11,29 +14,79 @@ function makeEnvironmentToggleValues(env = environments) {
   };
 }
 
+function getBootstrappedToggleValues() {
+  return {
+    facilityLocatorShowCommunityCares: true,
+  };
+}
+
+const initialToggleValues = {
+  ...makeEnvironmentToggleValues(),
+  ...getBootstrappedToggleValues(),
+};
+
+let currentToggleValues = initialToggleValues;
+const getToggleValues = () => currentToggleValues;
+
+async function connectFeatureToggle(
```

Comment:
This is the main function that's used. I decided to go with adding the feature flag values to the redux state because it allowed me to trigger a re-render when the values were received. 

Optimistically this will not impact the app visually since the feature that's being gated is not on the initial render. The component composition in the Facility Locator is kind of messy and would require some refactoring to enable a render blocking action.